### ISSUE TITLE
[Whitelist] Blur (42640)

### ISSUE THREAD

**Issue by pprotonrep** [ValveSoftware/Proton#119]
I tested [Blur](https://steamdb.info/app/42640/). After the usual shaders caching sluggishness at the beginning it runs beautifully.

- Ubuntu 18.04.1 LTS (64 bit 4.15.0-32-generic)
- GeForce GTX 1070/PCIe/SSE2 (396.51)
- Proton 3.7 (default configuration)

[System information from steam](https://gist.github.com/pprotonrep/43f7265a7457b17ceb5b848e89c26147)

**Comment by Zero86Sk** [comment]
Can conform it works, full speed.

[My Steam Information](https://gist.github.com/Zero86Sk/d629f657c1ae861c36657cdf238a7765)

**Comment by stephen304** [comment]
Just adding another confirmation on
* Arch Linux
* linux-next-git kernel
* using intel integrated graphics
* with an Xbox One S controller (via bluetooth)

Very smooth with the right graphics settings and my Xbox One S (bluetooth) controller works beautifully.

[steaminfo](https://gist.github.com/stephen304/c9e7528ae2d8bbf317022b58a1de970d)

**Comment by pprotonrep** [comment]
For the record: i noticed issues with two tracks: Brighton crashes the game on my system, an Tokio is so extremely bright, almost like the colours would be inverted. Otherwise smooth sailing

**Comment by stephen304** [comment]
@pprotonrep I do remember having some graphical bugs when playing before on windows, that brightness bug sounds familiar but the last time I went through a significant portion of the maps was ~5 years ago so I'll do some more testing.

**Comment by stephen304** [comment]
@pprotonrep Was the Brighton map in the proving grounds / Shannon section? I played through both Brighton maps there and didn't have any crash. Also, I played the Tokyo Shutoko map (third level) in city slickers / Khan section and the graphics seemed fine. Perhaps it's a graphics card related issue, what gfx do you have?

**Comment by pprotonrep** [comment]
GeForce GTX 1070

Actually i tested it a bit further: i only have problems with those two tracks that i mentioned, but only with specific cars... there might be some shaders conflicting or something (if that's even a thing :man_shrugging:). Thanks for your reply!

**Comment by stephen304** [comment]
@pprotonrep Yeah the game seems to have it's own issues as well. I remember in college when playing 4 way split screen it would crash randomly every maybe 10 or 20 rounds we played. It didn't seem to correlate with anything, but it wasn't bad enough to outweigh the fun we had playing since getting back in game is quick.

I've finally experienced the first crash through Proton, but it isn't consistent so it could be similar to the crashes on windows, some race condition or something (no pun intended). When starting the last level of the City Slicker section (the 1 on 1 race on Barcelona Gracia), it crashed as soon as the level starts (when the starting countdown hit 0.00). This level restricts you to 1 car, the Stallion Ford Mustang. Does the crash timing or car selection seem familiar to your testing by any chance?

**Comment by plafue** [comment]
All crashes are gone after using the PROTON_FORCE_LARGE_ADDRESS_AWARE environment variable
